build/mingw32.mingw.cmake

```diff
@@ -4,8 +4,8 @@
 
 set (CMAKE_SYSTEM_NAME "Windows")
 
-set (CROSS C:/Qt/Tools/mingw491_32)
-set (CROSSQT C:/Qt/5.4/mingw491_32)
+set (CROSS C:/Qt/Tools/mingw53_32)
```

Comment:
see above

Reply:
it doesn't seem too logical, but yes, mingw530_32 here and mingw53_32 there

Reply:
It is like 5.3.0 and 5.3 version.
I believe it is due to the agile approach. It leads to such small inconsistencies and less quality, but increases speed of development.